CMakeLists.txt

```diff
@@ -76,11 +75,6 @@ include(ev-project-bootstrap)
 
 ev_add_project()
 
-# create MF_ROOT_CA if not available
-if (ISO15118_2_GENERATE_AND_INSTALL_CERTIFICATES)
-    file(TOUCH config/certs/ca/mf/MF_ROOT_CA.pem)
```

Comment:
I think this file is not generated in the generate_certificates.py but its absence breaks OCPP

Reply:
Yes, thanks!  I need to check this, haven't tested yet, if the generated files are sufficient for running sil stuff.  But here I would rather at the generation of these necessary files into the generator script

Reply:
This should be mitigated with https://github.com/EVerest/libevse-security/tree/workaround/missing_certificates , which attempts to create the files that are configured but do not yet exist within the libevse-security. @AssemblyJohn